libparsec/crates/client/src/workspace/merge.rs

```diff
@@ -147,11 +147,34 @@ pub(super) fn merge_local_file_manifest(
 pub(super) fn merge_local_folder_manifest(
     local_author: DeviceID,
     timestamp: DateTime,
+    prevent_sync_pattern: Option<&Regex>,
     local: &LocalFolderManifest,
     remote: FolderManifest,
 ) -> MergeLocalFolderManifestOutcome {
+    // Destruct local and remote manifests to ensure this code with fail to compile
```

Comment:
Comment is broken (it talks about remote that is no longer destructed, and the 2nd line is missing !)

Reply:
For me it still deconstruct a remote manifest since I deconstruct the base manifest :sweat_smile:

Reply:
> For me it still deconstruct a remote manifest since I deconstruct the base manifest 😅

You're absolutely right ! I didn't see it :smile: 

(however there is still the second line of the comment to restore)

Reply:
done